objc/USearchObjective.mm

```diff
@@ -278,102 +325,134 @@ - (UInt32)searchHalf:(void const *_Nonnull)vector
 
 - (UInt32)getHalf:(USearchKey)key
            vector:(void *_Nonnull)vector
-            count:(UInt32)wanted {
+            count:(UInt32)wanted
+            error:(NSError**)error {
     std::size_t result = _native->get(key, (f16_t*)vector, static_cast<std::size_t>(wanted));
     return static_cast<UInt32>(result);
 }
 
-- (void)clear {
+- (void)clear:(NSError**)error {
     _native->clear();
 }
 
-- (void)reserve:(UInt32)count {
+- (void)reserve:(UInt32)count error:(NSError**)error {
     if (!_native->try_reserve(static_cast<std::size_t>(count))) {
-        @throw [NSException exceptionWithName:@"Can't reserve space"
-                                       reason:@"Memory allocation failed"
-                                     userInfo:nil];
+        *error = [NSError errorWithDomain:USearchErrorDomain
+                                     code:USearchAllocationError
+                                 userInfo:@{
+            NSLocalizedDescriptionKey: @"Can't reserve space",
+            NSLocalizedFailureReasonErrorKey: @"Memory allocation failed",
+        }];
     }
 }
 
-- (Boolean)contains:(USearchKey)key {
+- (Boolean)contains:(USearchKey)key error:(NSError**)error {
     return _native->contains(key);
 }
 
-- (UInt32)count:(USearchKey)key {
+- (UInt32)count:(USearchKey)key error:(NSError**)error {
     return static_cast<UInt32>(_native->count(key));
 }
```

Comment:
Is there no way to highlight those functions as `noexcept` in C++ terms? That why they probably still be accessed as computed properties of the class instance, right?

Reply:
I don't know. If you think the implementation of these will _never_ throw, and don't want to reserve the ability to throw in the API then I can drop these though.